_posts/01-06-01-Linux-Setup.md

```diff
@@ -0,0 +1,68 @@
+---
+isChild: true
+anchor:  linux_setup
+---
+
+## Linux Setup {#linux_setup_title}
+
+Most GNU/Linux distributions come with PHP available from the official repositories, but those packages usually are a little behind the current stable version. There are multiple ways to get newer PHP versions on such distributions. On Ubuntu and Debian-based GNU/Linux distributions, for instance, the best alternatives for native packages are provided and maintened by [Ondřej Surý][Ondrej Sury Blog], through his Personal Package Archive (PPA) on Ubuntu and DPA/bikeshed on Debian. Find instructions for each of these below. All that said, you can always use containers, compile the PHP Source code, etc.
+
+### Ubuntu-based distributions
+
+For Ubuntu distributions, the [PPA by Ondřej Surý][Ondrej Sury PPA] provides supported PHP versions along with many PECL extensions. To add this PPA to your system, perform the following steps in your terminal:
+
+1. First, add the PPA to your system's software sources using the command:
+
+   ```bash
+   sudo add-apt-repository ppa:ondrej/php
+   ```
+
+2. After adding the PPA, update your system's package list:
+
+   ```bash
+   sudo apt update
+   ```
+
+This will ensure that your system can access and install the latest PHP packages available in the PPA.
+
+#### Debian-based distributions
```

Comment:
Maybe just these headings should be sorted a bit. We could split distributions into two categories:

- Fedora based distributions
  - Add Remi's repository
- Debian based distributions (Ubuntu and derivatives)
  - Ondrey's repository
  
https://en.wikipedia.org/wiki/List_of_Linux_distributions

Reply:
I do think adding RHEL/Fedora dist instructions would be nice, but that's fine being a separate PR.